src/typeDefs/schema.graphql

```diff
@@ -92,6 +92,22 @@ type SpokenLanguage {
   nameNative: String!
 }
 
+type Submission {
+  id: ID!
+  submittedLink: String!
+  healthCareProfessionalName: String!
+  spokenLanguages: [SpokenLanguage]!
```

Comment:
@ermish We might want to let Submission have a list of strings for spokenLanguages in case users want to submit doctors that speak languages not included in the enum (even if search only allows EN/JP for MVP). Thoughts?